packages/fluentui/projects-test/src/rollup.ts

```diff
@@ -20,7 +20,7 @@ export async function rollup() {
 
   logger('STEP 1. Add dependencies to test project');
 
-  const rollupVersion = '2.7.3';
+  const rollupVersion = '2.68.0';
```

Comment:
`@fluentui/projects-test` explodes when `@fluentui/react-portal-compat-context` is transpiled with `swc` based on usage of `rollup/plugin-commonjs` (see error below). Setting the rollup version to at least `2.68.0` like the docs say fixed the issue:

![image](https://user-images.githubusercontent.com/8649804/224115161-04eecba2-3a3d-4b67-bdab-ed46b27bd72f.png)


**Error**:
```
verb @fluentui/projects-test test |  [!] (plugin commonjs) SyntaxError: Unexpected token (6:48) in /tmp/project-rollup--7012-JvE6lzYMysp0/test-app/node_modules/@fluentui/react-portal-compat-context/lib/PortalCompatContext.js
verb @fluentui/projects-test test |  node_modules/@fluentui/react-portal-compat-context/lib/PortalCompatContext.js (6:48)
verb @fluentui/projects-test test |  4: export const PortalCompatContextProvider = PortalCompatContext.Provider;
verb @fluentui/projects-test test |  5: export function usePortalCompat() {
verb @fluentui/projects-test test |  6:   return React.useContext(PortalCompatContext) ?? portalCompatContextDefaultValue;
verb @fluentui/projects-test test |                                                     ^
verb @fluentui/projects-test test |  7: }
verb @fluentui/projects-test test |  8: //# sourceMappingURL=PortalCompatContext.js.map
verb @fluentui/projects-test test |  SyntaxError: Unexpected token (6:48) in /tmp/project-rollup--7012-JvE6lzYMysp0/test-app/node_modules/@fluentui/react-portal-compat-context/lib/PortalCompatContext.js
verb @fluentui/projects-test test |      at Object.pp$4.raise (/tmp/project-rollup--7012-JvE6lzYMysp0/test-app/node_modules/rollup/dist/shared/rollup.js:3399:13)
verb @fluentui/projects-test test |      at Object.pp.unexpected (/tmp/project-rollup--7012-JvE6lzYMysp0/test-app/node_modules/rollup/dist/shared/rollup.js:1248:8)
verb @fluentui/projects-test test |      at Object.pp$3.parseExprAtom (/tmp/project-rollup--7012-JvE6lzYMysp0/test-app/node_modules/rollup/dist/shared/rollup.js:2822:10)
verb @fluentui/projects-test test |      at Object.parseExprAtom (/tmp/project-rollup--7012-JvE6lzYMysp0/test-app/node_modules/rollup/dist/shared/rollup.js:5836:77)
verb @fluentui/projects-test test |      at Object.pp$3.parseExprSubscripts (/tmp/project-rollup--7012-JvE6lzYMysp0/test-app/node_modules/rollup/dist/shared/rollup.js:2651:19)
verb @fluentui/projects-test test |      at Object.pp$3.parseMaybeUnary (/tmp/project-rollup--7012-JvE6lzYMysp0/test-app/node_modules/rollup/dist/shared/rollup.js:2628:17)
verb @fluentui/projects-test test |      at Object.parseMaybeUnary (/tmp/project-rollup--7012-JvE6lzYMysp0/test-app/node_modules/rollup/dist/shared/rollup.js:5720:29)
verb @fluentui/projects-test test |      at Object.pp$3.parseExprOps (/tmp/project-rollup--7012-JvE6lzYMysp0/test-app/node_modules/rollup/dist/shared/rollup.js:2572:19)
verb @fluentui/projects-test test |      at Object.pp$3.parseMaybeConditional (/tmp/project-rollup--7012-JvE6lzYMysp0/test-app/node_modules/rollup/dist/shared/rollup.js:2555:19)
verb @fluentui/projects-test test |      at Object.pp$3.parseMaybeAssign (/tmp/project-rollup--7012-JvE6lzYMysp0/test-app/node_modules/rollup/dist/shared/rollup.js:2528:19)
verb @fluentui/projects-test test |  
```